### PR TITLE
Create vercel-app.md

### PR DESCRIPTION
**Pull request by danieltprice**
Preview:
https://neon-next-git-dprice-expand-vercel-content-neondatabase.vercel.app/docs/guides/vercel-integration

- Broke up the content into separate sections
- Added a guide for the naturesnap application

### CODE REVIEW

File: content/docs/guides/vercel-app.md [hunk 1]
```diff
@@ -0,0 +1,378 @@
...
+- A [Vercel account](https://vercel.com/signup)
+- A [GitHub account](https://github.com/join)
+- A working installation of [psql](https://www.postgresql.org/download/)
+
+## Set up the application
+
+1. Download the example **naturesnap** project.
+
+    ```bash
+    wget https://github.com/neondatabase/naturesnap/archive/refs/heads/main.zip
```
Comment (duskpoet) [inline content/docs/guides/vercel-app.md:21]: why not git clone?

File: content/docs/guides/vercel-app.md [hunk 2]
```diff
@@ -0,0 +1,378 @@
...
+
+    ```bash
+    cd naturesnap
+    touch .env
+    ```
+
+    When finished, your `.env`  file should appear similar to the following (your hostname will differ).
+
+    ```text
+    DATABASE_URL=postgres://naturesnap:************@ep-snowy-water-747999.us-east-2.aws.neon.tech/naturesnap?sslmode=require&connect_timeout=0
```
Comment (duskpoet) [inline content/docs/guides/vercel-app.md:74]: I think for populating local envs we should advice users to user development branches

Comment (duskpoet) [reply content/docs/guides/vercel-app.md:74]: The point of the integration is so, that user doesn't need to manage neon project manually

**Comment by danieltprice** [comment]
Thank you for your feedback. We  decided to go a different direction with these docs in a new PR.

**Pull request closed without merging**